system-variables.md

```diff
@@ -270,8 +270,8 @@ For more possible values of this variable, see [Authentication plugin status](/s
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Type: Boolean
-- Default value: `OFF`
-- For compatibility, TiDB returns foreign key checks as `OFF`.
+- Default value: `ON`
```

Comment:
```suggestion
- Default value: `ON` (since v6.6.0)
```

Or is there a better way of showing when this changed? Now we only mark new variables with a version, but not important variable changes.